Add solver accounting section (#446)
cowprotocol/docs: docs/cow-protocol/reference/core/auctions/accounting.md

---
id: accounting
sidebar_position: 4
---

# Solver competition accounting process

> There is a formal and quite involved accounting process in place for the solver competition on each chain.  Currently, the execution of the accounting process takes place once a week for all chains the protocol operates on, and each accounting week starts on Tuesday at midnight UTC and concludes a week later at midnight UTC. This accounting process consists of the following:

Reviewer: ```suggestion
There is an accounting process in place for the solver competition. It is performed separately on each chain.  Currently, the execution of the accounting process takes place once a week for all chains the protocol operates on, and each accounting week starts on Tuesday at midnight UTC and concludes a week later at midnight UTC. This accounting process consists of the following:
```
